ckanext/canada/schemas/presets.yaml

```diff
@@ -3142,10 +3148,20 @@ presets:
       mimetype: application/vnd.google-earth.kmz
     - value: LAS
     - value: LYR
+    - value: Leapfrog Mining
+      replaces: [leapfrog mining]
+    - value: Leapfrog Hydro
+      replaces: [leapfrog hydro]
+    - value: Leapfrog Viewer
+      replaces: [leapfrog viewer]
     - label: MapInfo
       value: TAB
+    - value: MapInfo Grid
```

Comment:
we use the most common extension as the value for these choices (e.g. see TAB above for MapInfo)